proto/expr.proto

```diff
@@ -107,6 +107,8 @@ message ExprNode {
     SARG = 999;
     // Internal functions
     VNODE = 1101;
+    // Non-deterministic functions
+    NOW = 2022;
```

Comment:
Do we need NOW in expr proto? To my understanding, we should never need to serialize `NOW` expression as it should always be converted into a physical operator.

Reply:
> Do we need NOW in expr proto? To my understanding, we should never need to serialize `NOW` expression as it should always be converted into a physical operator.

It's @st1page 's suggestion. We can discuss on it.

Reply:
> we should never need to serialize NOW expression as it should always be converted into a physical operator.

You are right. But our frontend uses the same expression type enum in proto. So, to bind and plan it, we must add it in proto.